consensus/src/round_manager.rs

```diff
@@ -532,11 +534,13 @@ impl RoundManager {
         let expected_failed_authors = self.proposal_generator.compute_failed_authors(
             proposal.round(),
             proposal.quorum_cert().certified_block().round(),
+            proposal.is_nil_block(),
```

Comment:
it's always false, see https://github.com/aptos-labs/aptos-core/blob/274c3356f834bff5ee117f50c5cdbd41d3f2d8f7/consensus/consensus-types/src/proposal_msg.rs#L35